### PR TITLE
Prettier import order

### PR DESCRIPTION
**Pull request by afjlambert**

### CODE REVIEW

**Comment by afjlambert** [comment]
try it

**Comment by afjlambert** [comment]
@svenjens @robinpiets can you review?
Please merge.

File: apps/client/package.json [hunk 1]
```diff
@@ -72,12 +77,13 @@
     "@types/react-router-dom": "^5.1.5",
     "batch-promises": "0.0.3",
     "fast-xml-parser": "^3.17.4",
-    "history": "^5.0.0",
```
Comment (robinpiets) [inline apps/client/package.json:75]: You somehow removed the "history" package and it's still imported in `LocationPage.test.js`. That's why the tests are failing. Can you re-add?

`npm i --save history@5.0.0`

**Review by robinpiets — APPROVED** [review]
LGTM, only please fix the "history" package import

File: apps/client/package.json (hunk 1 above)
Comment (afjlambert) [reply apps/client/package.json:75]: He that's weird, I must have overlooked. Thanks

**Pull request merged**